bot/exts/moderation/infraction/infractions.py

```diff
@@ -280,8 +280,19 @@ async def unvoiceban(self, ctx: Context, user: FetchedMember) -> None:
 
     async def apply_mute(self, ctx: Context, user: Member, reason: t.Optional[str], **kwargs) -> None:
         """Apply a mute infraction with kwargs passed to `post_infraction`."""
-        if await _utils.get_active_infraction(ctx, user, "mute"):
-            return
+        if active := await _utils.get_active_infraction(ctx, user, "mute", send_msg=False):
+            if active["actor"] != self.bot.user.id:
+                await _utils.send_active_infraction_message(ctx, active)
+                return
+
+            # Let the current mute attempt override an automatically triggered mute.
+            log_text = await self.deactivate_infraction(active, notify=False)
+            if "Failure" in log_text:
+                await ctx.send(
+                    f":x: can't override infraction **mute** for {user.mention}: "
```

Comment:
This should be capitalized.

Reply:
Lowercase is consistent with the other messages sent out by infraction stuff.

Reply:
Are you sure? Here's from that same file..
https://github.com/python-discord/bot/blob/693573412dcf726426271c66f832e626fa77a5d4/bot/exts/moderation/infraction/infractions.py#L355
https://github.com/python-discord/bot/blob/693573412dcf726426271c66f832e626fa77a5d4/bot/exts/moderation/infraction/infractions.py#L155
https://github.com/python-discord/bot/blob/693573412dcf726426271c66f832e626fa77a5d4/bot/exts/moderation/infraction/infractions.py#L73

Reply:
Turns out there's no consistency. Whether this message is lower or upper-cased, it will be inconsistent with some other message. The confirmation messages use lower case, so that is what I was originally thinking of. I don't think making the messages consistent should be addressed by this PR.